config.js

```diff
@@ -0,0 +1,175 @@
+// NOTE: Neither of these files are committed and for good reason.
+//       You must provide your own.
+var token = require("./slack_token.js").token;
+var private_key = require("./service_account_key.js").key;
+
+var config = {
```

Comment:
not sure if I mentioned this before, but I like the way you have reorganized this.